starkinfra/pixchargeback/__pixchargeback.py

```diff
@@ -93,6 +95,7 @@ def query(limit=None, after=None, before=None, status=None, ids=None, flow=None,
     - before [datetime.date or string, default None]: date filter for objects created before a specified date. ex: datetime.date(2020, 3, 10)
     - status [list of strings, default None]: filter for status of retrieved objects. ex: ["created", "failed", "delivered", "closed", "canceled"]
     - ids [list of strings, default None]: list of ids to filter retrieved objects. ex: ["5656565656565656", "4545454545454545"]
+    - bacen_id [string]: unique transaction id returned from Central Bank. ex: "ccf9bd9c-e99d-999e-bab9-b999ca999f99"
```

Comment:
aqui vc passa um bacen_id só ou uma lista de bacen_id pra filtrar os resultados ??

Reply:
caso seja uma lista, vai ser necessario mudar a description

Reply:
outra coisa, ele é um parametro required ?? porq se nÃo for vc ter que colocar como "default None"

Reply:
Ids externos, não se faz listagem